### PR TITLE
Updated the locale text for the Spanish culture

### PR DESCRIPTION
**Pull request by balajiSF3690**
Updated the locale text for the Spanish culture

### CODE REVIEW

File: src/es.json [hunk 1]
```diff
@@ -251,37 +251,37 @@
...
+            "MergeCells": "Combinar celdas",
             "MergeAll": "Fusionar todo",
             "MergeHorizontally": "Fusionar horizontalmente",
             "MergeVertically": "Fusionar verticalmente",
             "Unmerge": "Desunir",
-            "UnmergeCells": "Unmerge Cells",
-            "SelectMergeType": "Seleccionar tipo de fusión",
-            "MergeCellsAlert": "La combinación de celdas solo conservará el valor superior izquierdo (superior). Fusionar de todos modos?",
+            "UnmergeCells": "Desunir Celdas",
+            "SelectMergeType": "Seleccionar tipo de combinación",
```
Comment (PiramanayagamSyncfusion) [inline src/es.json:260]: Use this "Seleccione el tipo de combinación"

File: src/es.json [hunk 2]
```diff
@@ -876,28 +876,28 @@
             "customRange": "Rango personalizado"
         },
         "timepicker": {
-            "placeholder": "Elige un momento"
+            "placeholder": "Elegir fecha"
```
Comment (PiramanayagamSyncfusion) [inline src/es.json:879]: Use this "Elegir un fecha"

File: src/es.json [hunk 3]
```diff
@@ -1000,19 +1000,19 @@
...
-            "weeks": "Semanas)",
-            "months": "Meses)",
-            "years": "Años)",
+            "days": "Día(s)",
+            "weeks": "Semana(s)",
+            "months": "Mes(es)",
+            "years": "Año(s)",
             "every": "cada",
-            "summaryTimes": "veces)",
+            "summaryTimes": "veces",
```
Comment (PiramanayagamSyncfusion) [inline src/es.json:1008]: Use this "vece(s)"

File: src/es.json [hunk 4]
```diff
@@ -1110,8 +1110,8 @@
             "taskType": "Tipo de tarea",
             "unassignedTask": "Tarea no asignada",
             "group": "Grupo",
-            "indent": "Sangrar",
-            "outdent": "Outdent",
+            "indent": "Sangría",
+            "outdent": "Anular sangría",
```
Comment (PiramanayagamSyncfusion) [inline src/es.json:1114]: Use this "Eliminar sangría" which means remove the indent

File: src/es.json [hunk 5]
```diff
@@ -1110,8 +1110,8 @@
             "taskType": "Tipo de tarea",
```
Comment (PiramanayagamSyncfusion) [inline src/es.json:1110]: Use this.
"timelineCell": "Celda de la línea de tiempo",

Comment (balajiSF3690) [reply src/es.json:1110]: Changed

File: src/es.json (hunk 1 above)
Comment (balajiSF3690) [reply src/es.json]: Changed

File: src/es.json [hunk 6]
```diff
@@ -627,8 +627,8 @@
             "FourArrowsGray": "4 flechas (coloreadas)",
             "FiveArrowsColor": "5 flechas (gris)",
             "FiveArrowsGray": "5 flechas (coloreadas)",
-            "ThreeTrafficLights1": "3 semáforos (sin marco)",
-            "ThreeTrafficLights2": "3 semáforos (bordeados)",
+            "ThreeTrafficLights1": "3 Semáforos (Sin Borde)",
+            "ThreeTrafficLights2": "3 semáforos (con borde)",
```
[the remark this answers is not in the document]
Comment (balajiSF3690) [reply src/es.json]: Changed

File: src/es.json (hunk 2 above)
Comment (balajiSF3690) [reply src/es.json]: Changed

File: src/es.json (hunk 3 above)
Comment (balajiSF3690) [reply src/es.json]: Changed

File: src/es.json (hunk 4 above)
Comment (balajiSF3690) [reply src/es.json]: Changed

**Review by PiramanayagamSyncfusion — APPROVED** [review]
Changes fine

**Pull request merged**